repro.in

```diff
@@ -169,14 +146,8 @@ check_pkgbuild() {
 
 build_package(){
     local build=$1
-    local bind_builddir=$2
-    local builddir=${3:-"/build"}
-    nspawn_opts=""
-    if (( bind_builddir )); then
-        nspawn_opts="--bind=$PWD:$builddir"
-    fi
+    local builddir=${2:-"/build"}
     exec_nspawn $build \
-        $nspawn_opts \
```

Comment:
A bit unsure in this bit, can't test at the moment as this tool fails for me with some issues in buildinfo, taking a look at it next.